decko/lib/decko/swagger.rb

```diff
@@ -0,0 +1,46 @@
+module Decko
+  # hacky first go at swagger generation.
+  #
+  # In decko, it really just converts yaml to ruby and back again.
+  #
+  # But it's useful for generating decko swagger docs.
+  class Swagger
```

Comment:
At this point, the only thing the ruby representation buys us is a mechanism to merge wikirate and decko yaml.  But it's a start...